packages/pigeon/tool/shared/generation.dart

```diff
@@ -20,10 +20,20 @@ enum GeneratorLanguage {
   swift,
 }
 
+const Set<GeneratorLanguage> allButSwiftAndDart = <GeneratorLanguage>{
+  GeneratorLanguage.cpp,
+  GeneratorLanguage.gobject,
+  GeneratorLanguage.java,
+  GeneratorLanguage.objc,
+};
+
 // A map of pigeons/ files to the languages that they can't yet be generated
 // for due to limitations of that generator.
 const Map<String, Set<GeneratorLanguage>> _unsupportedFiles =
-    <String, Set<GeneratorLanguage>>{};
+    <String, Set<GeneratorLanguage>>{
+  'event_channel_tests': allButSwiftAndDart,
+  'proxy_api_tests': allButSwiftAndDart,
```

Comment:
I'm not sure factoring out a shared constant is actually useful here, given that we could easily re-diverge them (e.g., by adding event channel support for Windows and Linux).

Reply:
Guess I expected it might be used more than this, but fair point.